code/game/objects/effects/overlays.dm

```diff
@@ -508,9 +510,9 @@
 	icon = 'icons/mob/mob.dmi'
 	duration = 15
 
-/obj/effect/overlay/temp/dust_animation/New(loc, dust_icon)
+/obj/effect/overlay/temp/dust_animation/Initialize(mapload, dust_icon)
 	icon_state = dust_icon // Before ..() so the correct icon is flick()'d
-	..()
```

Comment:
See comment

This was running code before parent New() that would now happen after New().

Would the behavior be correct now that its running that code after new but before initialize?